File: java/anagram.java
import java.util.Arrays;
public class anagram{
    public static void main(String args[]){
        String str1 = " silent";
        String str2 = "listen";
        if (str1.length()==str2.length()){
            char[] ar1 = str1.toCharArray();
            Arrays.sort(ar1);
            System.out.println(Arrays.toString(ar1));
            char[] ar2 = str2.toCharArray();
            Arrays.sort(ar2);
            System.out.println(Arrays.toString(ar2));
            System.out.println(Arrays.equals(ar1,ar2));
            if (ar1.equals(ar2)){
                System.out.println("string are anagram");
            }
            else{
                System.out.println("not an anagram");
            }

        }
    }
    }

File: python/anagram.py
def anagram(str1,str2):
    if sorted(str1)==sorted(str2):
        print("Given string are anagram")
        
    else:
        print("Not an anagram!! Try again")
while True:
    str1 = input('enter your first string\t\n')
    str2 = input('enter your second string\t')
    anagram(str1,str2)

